assets/humans.txt

```diff
@@ -88,6 +87,7 @@ Access:
  Tomi Lahdesmaki
  Chloe Guest
  Dave Thompson
+ Alice Fuzier-Cayla
```

Comment:
Minor: this list is sorted by reverse order of leaving date – Alice should be at the top because she left most recently.